collections/projects.go

```diff
@@ -37,12 +43,31 @@ func (p *Projects) GetStoreID() *uuid.UUID {
 }
 
 func (p *Projects) Create(ctx context.Context, name, scope, fcWalletAddress string) (*Project, error) {
+	// Create subdomain for the project
+	domain := ""
+	records := []*dns.Record{}
+	if p.dnsManager.Started {
+		safesd, err := dns.CreateURLSafeSubdomain(name, 8)
+		if err != nil {
+			return nil, err
+		}
+		rds, err := p.dnsManager.NewDNSLink(safesd, defaultHash)
```

Comment:
went with DNS link to start. probably a good one to do all time. we can run the UpdateRecord method to change the dnslink has pointer preeeetty nicely. currently it defaults to the 404 page (as a hash), but we can create a similar style one for an empty project